s3-bucket/vars.tf

```diff
@@ -0,0 +1,17 @@
+variable "bucket-name" {
+  default     = "flakyc00ki3s"
+  type        = string
+  description = "the s3 bucket name"
+}
+
+variable "tag-name" {
```

Comment:
Although typically useful, I would probably drop the tags just to keep this terrraform minimal